### DISCUSSION TITLE
Question regarding new rule making field BT-746 "Winner Listed" mandatory for winners.

### DISCUSSION THREAD

**Discussion by Fynnyan** [OP-TED/eForms-SDK#730] [Q&A] [answered]
Hello

I got a question regarding a new validation rule we encountered for field BT-746 - Winner Listed (`efbc:ListedOnRegulatedMarketIndicator`)

Since the version `1.8` the field is newly reported as a mandatory for winner organizations when creation a contract award notice (verfied for forms 29/30). In previous versions omitting the field for winners did not trigger a validation error.

In the schema documentation, it is stated that the field can be omitted.

> The element "efbc:ListedOnRegulatedMarketIndicator" must either be omitted for all Tenderers and subcontractors, or included for all Tenderers and subcontractors. 
>
> https://docs.ted.europa.eu/eforms/latest/schema/parties.html#winnerTendererSection

Also in the "[Regulation (EU) 2022/2303](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32022R2303)", which the the eForms implement, it is declared as optional.

I also found no mention of the rule change in the release notes.

Is this a bug or a change to the rules that is not properly documented?

**Comment by YvesJo** [comment, marked as answer]
Hi,
This paragraph has unfortunately remained in its initial statement while it should have been updated since.
Some clarifications were required regarding organizations because the Regulation Annex does not account precisely for the various roles/sub-roles required information. These clarifications lead to the detailed tables on that same page.
Depending on the type of winning organization (listed on a regulated market, natural person, privately owned) the set of information to specify differ. (e.g. UBO for privately owned companies, not for the ones on a regulated market and not for natural persons)
To make sure the winning organization classification is not the result of a forgotten field, BT-746 had to be made mandatory (and BT-633 mandator if BT-746 is set to false), allowing this way to identify the correct category the organization falls into and the subsequent requirements to be applied to. This rule was prepared a long time ago and reactivated with SDK 1.8 as part of the progressive set of rules activation.
All this information was deemed necessary as part of the "know your customer" and "anti-money laundering" legislation.
Also the rules with UBOs have been loosened compared to what was initially planned and BT-633 is now only made optional when BT-746 is set to false, keeping a sequence in the logic and still preventing the presence of conflicting information.
